Cargo.toml

```diff
@@ -15,6 +15,8 @@ serde = "1.0"
 serde_json = "1.0"
 cpython = { version = "0.1", default-features = false }
 cpython-json = { version = "0.2", default-features = false }
+error-chain = { version = "0.11.0", optional = true }
 
 [features]
 default = ["cpython/python3-sys"]
+errorchain = ["error-chain"]
```

Comment:
Instead of listing a separate feature named `errorchain`, you can just keep `error-chain` optional and use `#[cfg(feature = "error-chain")]` throughout your code.

Reply:
Didn't know you could use an optional crate name directly as a feature !